server/src/repositories/asset-job.repository.ts

```diff
@@ -343,4 +343,9 @@ export class AssetJobRepository {
       .orderBy('assets.createdAt', 'desc')
       .stream();
   }
+
+  @GenerateSql({ params: [DummyValue.DATE], stream: true })
+  streamForMigrationJob() {
+    return this.db.selectFrom('assets').select(['id']).where('assets.deletedAt', 'is', sql.lit(null)).stream();
```

Comment:
Why do you need `sql.lit` here? Everywhere else we're doing this `.where('assets.deletedAt', 'is', null)` just works

Reply:
True, I have seen it sometimes still put a parameter instead of the literal in the generated SQL and this doesn't. `null` seems to work without it though.